src/PointwiseFunctions/GeneralRelativity/ComputeGhQuantities.hpp

```diff
@@ -715,5 +722,285 @@ struct TraceExtrinsicCurvatureCompute
       const tnsr::II<DataVector, SpatialDim, Frame>&) = &trace;
   using base = gr::Tags::TraceExtrinsicCurvature<DataVector>;
 };
+
+// @{
+/*!
+ * \brief Compute item to compute constraint-damping parameters for a single-BH
+ * evolution.
+ *
+ * \details Can be retrieved using
+ * `GeneralizedHarmonic::Tags::ConstraintGamma0`,
+ * `GeneralizedHarmonic::Tags::ConstraintGamma1`, and
+ * `GeneralizedHarmonic::Tags::ConstraintGamma2`.
+ */
+template <size_t SpatialDim, typename Frame>
+struct ConstraintGamma0Compute : ConstraintGamma0, db::ComputeTag {
+  using argument_tags = tmpl::list<::Tags::Coordinates<SpatialDim, Frame>>;
+  static auto function(
+      const tnsr::I<DataVector, SpatialDim, Frame>& coords) noexcept {
+    const DataVector r_squared = get(dot_product(coords, coords));
+    Scalar<DataVector> gamma = make_with_value<type>(coords, 0.);
+    get(gamma) = 3. * exp(-0.0078125 * r_squared) + 0.001;
+    return gamma;
+  }
+  using base = ConstraintGamma0;
+};
+
+template <size_t SpatialDim, typename Frame>
+struct ConstraintGamma1Compute : ConstraintGamma1, db::ComputeTag {
```

Comment:
Here too for some reason only the first class is being documented.

Reply:
Doc for these also compiles fine for me, see [screenshot here](https://github.com/sxs-collaboration/spectre/pull/1444#discussion_r286612777)

Reply:
see [here](https://github.com/sxs-collaboration/spectre/pull/1444#discussion_r286635875)